.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "node": true,
+    "es2020": true
+  },
+  "extends": ["@mscharley"],
+  "parser": "@typescript-eslint/parser",
```

Comment:
Setting parser explicitly isn't necessary with my ruleset and can be actively harmful in some cases.